wikidata_game/MatchesRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ReferenceIsland;
```

Comment:
The ns should prbably be more specific here, as if this pattern is going to be used, It will not be the only thing under `ReferenceIsland`

Reply:
As far as I can tell if we moved this to a folder e.g. "DataAccess" then we would put in a "DataAccess" subnamespace name but given that right now (perhaps wrongly) this class is in the top level namespace this is ok